website/docs/automation/gitops/argocd/update_application.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Updating an application"
+sidebar_position: 40
+---
+
+Now we can use Argo CD and Kustomize to deploy patches to our application manifests using GitOps
+For example, lets increase the number of `replicas` for `ui` deployment to `3`
+
+```kustomization
+automation/gitops/argocd/update-application/deployment-patch.yaml
+Deployment/ui
+```
+
+Copy patch file to the Git repository directory:
+
+```bash
+$ cp /workspace/modules/automation/gitops/argocd/update-application/deployment-patch.yaml ~/environment/gitops/apps/deployment-patch.yaml
+```
+
+You can review planned changes in the file `apps/deployment-patch.yaml`
+
+To apply the patch edit the file `apps/kustomization.yaml` like in the example below:
+
+```file
+automation/gitops/argocd/update-application/kustomization.yaml
```

Comment:
```
$ cat apps/kustomization.yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
- ui
patches:
- deployment-patch.yaml
```
This relates to the `- ui` line being different in this diff.

Reply:
I changed approach and now it shows the same correct content like in the file which is used for deployment.